app/controllers/user_sessions_controller.rb

```diff
@@ -1,4 +1,6 @@
 class UserSessionsController < ApplicationController
+
+  before_filter :require_no_user, :only => [:new]
```

Comment:
`before_filter` was used in Rails 3 and is updated to `before_action` in rails 4 and before_filter is deprecated in rails 5 and is removed from rails 5.1 so i think it would be better to use `before_action` instead of `before_filter`

Reply:
Otherwise this looks good!

Reply:
Looks good!! And a great catch too!! :+1:

Reply:
Yeah for sure. Pushing it now